### ISSUE TITLE
Nodes contain invalid inputs

### ISSUE THREAD

**Issue by Chrono9084** [LukeyJohnson/BakeryForDynamo#6]
Since the newest version of Dynamo launched with the most recent Revit 2020 hotfix, this error has popped up with multiple packages (including yours).

![image](https://user-images.githubusercontent.com/37596510/64433111-755f8c00-d083-11e9-9591-b50f15045b90.png)
![image](https://user-images.githubusercontent.com/37596510/64433205-a049e000-d083-11e9-917b-eb469a639482.png)

**Comment by LukeyJohnson** [comment]
Yes Bakery needs to be migrated and fixed for 2020 / Dynamo 2.1.

________________________________
